content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -144,36 +172,40 @@ Client: I am doing this for the first time
 Client: BYE
 Closing connection
 ```
-### Client-side Programming
+### Programming on the Client-side
 #### Establish a Socket Connection
-A Socket instance can be obtained in two ways: 
-    
-1. Server side receives it as a return value of the accept method.
-2. You can instantiate it as a Socket like so:
+There are two ways to get a Socket instance:
+
+1.Server receives the accept method as a return value.
+2. You can use the following code to create a Socket:
+
 ```java
 socket = new Socket("localhost", 5000);
 ```
-localhost - is a domain name like google.com, which takes you to your computer. It is resolved to IP address 127.0.0.1.
-Additionally, the second argument is a port number.
-Like before we create I/O streams via the socket object:
+Localhost is a domain name that redirects you to your computer, similar to google.com. It resolves to 127.0.0.1 as the IP address.
+A port number is also included in the second argument.
+We generate I/O streams using the socket object, just as we did before:
+
 ```java
 inputStreamReader = new InputStreamReader(socket.getInputStream());  
 outputStreamWriter = new OutputStreamWriter(socket.getOutputStream());
 ``` 
 Client-side, we need to use a Scanner object to get input from the user from the console/terminal. 
+
 ```java
 Scanner scanner = new Scanner(System.in);
 ```
-`System.in` specifies that we are getting the keyboard input.
+`System. in` specifies that we are getting the keyboard input.
```

Comment:
I realized I may have confused you here. I know I said to add a space after the period before a new word, but this wouldn't apply to code. So change:

>`System. in`

To:

>`System.in`

removing the space between **System.** and **in**. Sorry for the confusion!